manageiq-operator/Dockerfile

```diff
@@ -1,5 +1,18 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.23 as builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal as builder
+
+COPY go.mod /go.mod
+
+RUN microdnf -y install gzip jq tar && \
+    arch=$(uname -m) && \
```

Comment:
I cannot find the download link https://go.dev/dl/ if the platform is
ppc64le 
s390x

Wonder if you can just use amd64 instead for both power and s390x.

Reply:
https://go.dev/dl/go1.23.2.linux-ppc64le.tar.gz
https://go.dev/dl/go1.23.2.linux-s390x.tar.gz

Reply:
Forgot to expand the tab. Looks good. Thanks